scripts/ingests/bones_archive/ingest_bones_sources.py

```diff
@@ -0,0 +1,155 @@
+from simple.schema import *
+from simple.schema import REFERENCE_TABLES
+from astrodb_utils import (
+    load_astrodb,
+    AstroDBError,
+)
+from astrodb_utils.sources import ingest_names, ingest_source, find_source_in_db
+from astrodb_utils.publications import ingest_publication, find_publication
+
+import sys
+from astrodb_utils.utils import logger
+sys.path.append(".")
+from astropy.io import ascii
+
+# logger = logging.getLogger(__name__)
+names_ingested = 0
+sources_ingested = 0
+skipped = 0
+total = 0
+already_exists = 0
+multiple_sources = 0
+no_sources = 0
+inside_if = 0
+
+
+DB_SAVE = False
+RECREATE_DB = True
+db = load_astrodb("SIMPLE.sqlite", recreatedb=RECREATE_DB, reference_tables=REFERENCE_TABLES)
+
+ingest_publication(db, bibcode="2018MNRAS.479.1383Z", reference="Zhan18.1352")
+ingest_publication(db, bibcode="2018MNRAS.480.5447Z", reference="Zhan18.2054")
+
+link = (
+    "scripts/ingests/bones_archive/theBonesArchivePhotometryWithADS.csv"
+)
+
+# read the csv data into an astropy table
+bones_sheet_table = ascii.read(
+    link,
+    format="csv",
+    data_start=1,
+    header_start=0,
+    guess=False,
+    fast_reader=False,
+    delimiter=",",
+)
+
+def extractADS(link):
+    start = link.find("abs/") + 4
+    end = link.find("/abstract")
+    ads = link[start:end]
+    ads = ads.replace("%26", "&")
+    logger.debug(f"ads: {ads}")
+    return ads
+
+
+for source in bones_sheet_table[:60]:
+    bones_name = source["NAME"].replace("\u2212", "-")
+    match = None
+
+    if len(bones_name) > 0 and bones_name != "null":
+        match = find_source_in_db(
+            db,
+            bones_name,
+            ra=source["RA"],
+            dec=source["DEC"],
+            ra_col_name="ra",
+            dec_col_name="dec",
+        )
+        if len(match) == 1:
+            try:
+                ingest_names(
+                    db, match[0], bones_name
+                )  # ingest new names while were at it
+                names_ingested += 1
+            except AstroDBError as e:
+                raise e  # only error is if there is a preexisting name anyways.
+
+    if match is None:
+        match = find_source_in_db(
+            db,
+            source["NAME"],
+            ra=source["RA"],
+            dec=source["DEC"],
+            ra_col_name="ra",
+            dec_col_name="dec",
+        )
+
+    if len(match) == 0:
+        # ingest_publications for the ADS link
+        ads = extractADS(source["ADS_Link"])
+        adsMatch = None
+        adsRef = source["Discovery Ref."]
+        adsMatch = find_publication(
+            db,
+            bibcode = ads
+        )
+        logger.debug(f"adsMatch: {adsMatch}")
+
+        if not adsMatch[0]:
+            logger.debug(f"ingesting publication {ads}")
+            ingest_publication(
+                db,
+                bibcode = ads,
+                reference=adsRef
```

Comment:
```suggestion
```
This is the line which is giving the inconsistent formatting for the references.